charts/gateway-helm/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml

```diff
@@ -149,8 +149,8 @@ spec:
                                       of compute resources required. If Requests is
                                       omitted for a container, it defaults to Limits
                                       if that is explicitly specified, otherwise to
-                                      an implementation-defined value. More info:
-                                      https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
+                                      an implementation-defined value. Requests cannot
```

Comment:
should change `api/config/v1alpha1/shared_types.go:83`,  otherwise `make manifests` will override this.

Reply:
if so, the gencheck will fail?

Reply:
related to an upstream comment update